src/harness/fourslash.ts

```diff
@@ -391,7 +391,7 @@ module FourSlash {
             this.currentCaretPosition = pos;
 
             var lineStarts = ts.computeLineStarts(this.getCurrentFileContent());
-            var lineCharPos = ts.getLineAndCharacterOfPosition(lineStarts, pos);
+            var lineCharPos = ts.computeLineAndCharacterOfPosition(lineStarts, pos);

```

Comment:
`computeLineAndCharacterOfPosition` accepts lineMap as `number[]` and use it to compute line and character. `getLineAndCharacterOfPosition` accepts `SourceFile`, picks lineMap from the file (and initializes it if it was not yet created) and then invokes `computeLineAndCharacterOfPosition`